pkg/database/mysql/mysql.go

```diff
@@ -36,3 +37,9 @@ func (m *MySQL) Name() string {
 func (m *MySQL) Namespace() string {
 	return m.cluster.Namespace
 }
+
+func IsMySQL(obj client.Object) bool {
```

Comment:
This code is not used but we discussed it with @egegunes  and it will be removed in another PR.